### PR TITLE
Clean up Ubuntu and Alma directories

### PR DESCRIPTION
**Pull request by LiquidPT**
This cleans up extra files within the Ubuntu and Alma directories, consolidating functionality within each of those as much as possible. 

Then next step will be to find commonalities between the distros that can be moved to /common

I do not know what the changes to the 3 MD files in root are. When I changed laptops this month, it would not let me discard changes, nor could it show me any visible changes. The comparison here also doesn't show any changes.

### CODE REVIEW

File: alma/common/hpc-tuning.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
 # Disable some unneeded services by default (administrators can re-enable if desired)
+systemctl disable wpa_supplicant
```
Comment (yakovdyadkin) [inline alma/common/hpc-tuning.sh:4]: Just curious why the additional services need to be disabled?

**Comment by yakovdyadkin** [comment]
Do you think the 3 MD file changes are related to line endings, considering line endings are now explicitly set to LF in gitattributes?

**Review by yakovdyadkin — APPROVED** [review]

File: alma/common/hpc-tuning.sh (hunk 1 above)
Comment (LiquidPT) [reply alma/common/hpc-tuning.sh:4]: > Just curious why the additional services need to be disabled?

this was one of my questions for you about the differences in the existing files. I took this from the existing alma 8.x hpc-tuning.sh

**Comment by LiquidPT** [comment]
> Do you think the 3 MD file changes are related to line endings, considering line endings are now explicitly set to LF in gitattributes?

Maybe, though I added tried removing the .gitattributes file and it should let me undo it as well. If it is line0endings, it's probably good to standardize on linux-style anyways.

**Comment by LiquidPT** [comment]
> Do you think the 3 MD file changes are related to line endings, considering line endings are now explicitly set to LF in gitattributes?

Maybe, though I added tried removing the .gitattributes file and it should let me undo it as well. If it is line0endings, it's probably good to standardize on linux-style anyways.

**Comment by LiquidPT** [comment]
> Do you think the 3 MD file changes are related to line endings, considering line endings are now explicitly set to LF in gitattributes?

Maybe, though I added tried removing the .gitattributes file and it should let me undo it as well. If it is line0endings, it's probably good to standardize on linux-style anyways.

**Pull request merged**